pkg/util/cluster.go

```diff
@@ -190,3 +191,14 @@ func IsClusterIdentifyUnique(controlPlaneClient karmadaclientset.Interface, id s
 	}
 	return true, "", nil
 }
+
+// IsClusterSpecPartCriticalUpdated checks whether the critical part of ClusterSpec has been updated
+func IsClusterSpecPartCriticalUpdated(newSpec, oldSpec clusterv1alpha1.ClusterSpec) bool {
```

Comment:
The name `ClusterSpecPartCritical` seems not straightforward enough, please consider to remove this function, or rename it to `ClusterAccessCredentialChanged`.

Reply:
fixed